woocommerce/wc-admin
# Add CES survey for importing products

This PR adds CES survey trigger on importing products.

Partially fixes https://github.com/woocommerce/woocommerce-admin/issues/6307.

### Detailed test instructions:

- Make sure tracking is enabled in settings: 
```
/wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com
```
- Delete the option `woocommerce_ces_shown_for_actions` to make sure CES prompt triggers when updating settings.
- Enable the logging of Tracks events to your browser dev console:
```
localStorage.setItem( 'debug', 'wc-admin:tracks' );
```

- If you don't have a product CSV export, you can obtain a sample CSV [here](https://gist.githubusercontent.com/ilyasfoo/507f9579531cf4bf50fe4c0e9c48a23d/raw/05e47e6731471464c757e893c3f2d8a9b89453c0/product-export.csv).
- Go to Products > All Products.
- Click on "Import".
- Upload CSV file and finish the import process.
- Observe CES prompt "How easy was it to import products?" is displayed.
